qml/EpisodeItem.qml

```diff
@@ -93,7 +93,7 @@ ListItem {
                 icon.source: 'image://theme/icon-m-add'
 				onClicked: {
                     player.enqueueEpisode(id, function () {
-						if (player.episode==0) {
+                        if (player.episode === 0) {
```

Comment:
hehe this one is a can of worms....
I guess that since the whole project uses spaces it should also, though I have been convinced we should really be using tabs for accessibility by visually impaired programmers.

https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/

Probably a future goal to convert all files to tabs.

Reply:
when the PRs are closed it would be a good time to run sed and start fresh with tabs everywhere... Maybe we can configure sailfishIDE via the repo as well